config-schema/config.json

```diff
@@ -756,23 +756,7 @@
           "tdrDataLoadHeader": "RelatedMaterial"
         }
       ],
-      "downloadFilesOutputs": [
-        {
-          "domain": "MetadataDownloadTemplate",
-          "columnIndex": 19,
-          "editable": true
-        },
-        {
-          "domain": "MetadataReviewDetailTemplate",
-          "columnIndex": 21,
-          "editable": true
-        },
-        {
```

Comment:
Can you retain the `BagitExportTemplate` domain. Whether a field is included in the Bagit is controlled by the `allowExport` property.